nototools/noto_cmap_reqs.py

```diff
@@ -3285,10 +3285,10 @@ def _dump_primaries():
   for block in unicode_data.block_names():
     block_range = unicode_data.block_range(block)
     primary_script = _primary_script_for_block(block)
-    print '%13s %6s %s' % (
+    print('%13s %6s %s\n' % (
```

Comment:
It seems the '\n' is not wanted here?  Print already issues a newline, right?

Reply:
You  are right! Sorry!